Sources/BraintreeCore/BTCoreConstants.swift

```diff
@@ -15,6 +15,9 @@ import Foundation
 
     /// URL Scheme for PayPal App
     public static let payPalURLScheme: String = "paypal-app-switch-checkout"
+    
+    /// Universal link for PayPal app (app switch flow)
+    public static let payPalUniversalLink: String = "https://paypal.com/app-switch-checkout"
```

Comment:
This will always return true regardless of if the app is installed or not. You can test this by doing the following in simulator without the PayPal app installed:
```swift
print(UIApplication.shared.canOpenURL(URL(string: "https://paypal.com/app-switch-checkout")!))
```
This is a valid URL string so it's always possible to open it.